topics/transcriptomics/tutorials/scrna-raceid/tutorial.md

```diff
@@ -256,9 +271,9 @@ The bottom row shows the count distributions of the Library Size and Number of F
 
 > ### {% icon details %} Details: Why the Same Number of Features
 >
-> * RaceID normalises the data so that all cells are compared using the same features. If the features compared between cells are different, then it is hard to make a meaningful assessment of how much one cell differs from another. The features that are selected need to be *meaningful*, meaning that they must describe or contribute to the biological variation in the data, and therefore must be differentially expressed between cells.
+> * RaceID normalizes the data to compare all cells using the the same features, as it is difficult to make a meaningful comparison comparison otherwise. Selected features must be *meaningful*, describing or contributing to the biological variation in the data, and therefore must be differentially expressed between cells.
```

Comment:
```suggestion
> * RaceID normalizes the data to compare all cells using the the same features, as it is difficult to make a meaningful comparison otherwise. Selected features must be *meaningful*, describing or contributing to the biological variation in the data, and therefore must be differentially expressed between cells.
```

Reply:
good catch :+1: